pkg/kubelet/kubelet.go

```diff
@@ -110,6 +111,18 @@ const (
 	housekeepingPeriod = time.Second * 2
 
 	etcHostsPath = "/etc/hosts"
+
+	// Capacity of the channel for recieving pod lifecycle events. This number
+	// is a bit arbitrary and may be adjusted in the future.
+	plegChannelCapacity = 1000
+
+	// TODO(yujuhong): migrate some of the parameters to kubelet flags.
+	// Relisting is used to discover missing container events.
+	// Use a shorter period because generic PLEG relies on relisting for
+	// discovering container events.
+	plegRelistPeriod = time.Second * 3
```

Comment:
This is an implementation detail. We can have a flag just for the stabilization phase maybe, but I'd prefer not cutting a release with the flag. 
Moreover, once we centralize all the pod interactions, the re-listing period should not be of concern right?